### PR TITLE
Watch for the magic!

### PR DESCRIPTION
**Pull request by rgee0**
it fixes an open issue:  [issue-1](#1)

### CODE REVIEW

File: FRED [hunk 1]
```diff
@@ -1 +1,2 @@
 rgee0
```
Comment (rgee0) [inline FRED:1]: this

**Comment by rgee0** [comment]
Derek set title:

**Comment by rgee0** [comment]
Derek set title: This is a splendid title

**Comment by rgee0** [comment]
Derek set title: Watch for the magic!

**Comment by rgee0** [comment]
Derek lock

**Comment by rgee0** [comment]
Derek lock

**Comment by rgee0** [comment]
Derek unlock

**Comment by rgee0** [comment]
Derek close

**Comment by rgee0** [comment]
Derek reopen

**Comment by alexellis** [comment]
Derek close

**Pull request merged**